activerecord/test/cases/schema_dumper_test.rb

```diff
@@ -902,6 +902,28 @@ def down
       $stdout = original
     end
   end
+
+  def test_schema_dump_keeps_enum_intact_if_it_contains_comma
+    original, $stdout = $stdout, StringIO.new
```

Comment:
Should we use [`capture_io`](https://github.com/minitest/minitest/blob/d84437f874b42be1c3d46b81640904144fc7dac4/lib/minitest/assertions.rb#L538)?

Reply:
I used it to suppress the output of the up and down migrations, as I saw in other examples. The output of the dump is already being captured with `capture_io`.

Maybe it's better to remove this line entirely?